base/abstractarray.jl

```diff
@@ -2015,9 +2016,6 @@ julia> hvcat((2,2,2), a,b,c,d,e,f)
  3  4
  5  6
```

Comment:
```suggestion
 5  6
 
julia> hvcat((2,2,2), a,b,c,d,e,f) == hvcat(2, a,b,c,d,e,f)
true
```
? Not sure, because the doctests are already long, but this could demonstrate the single-integer usage.